walletunlocker/service_test.go

```diff
@@ -442,11 +418,7 @@ func TestChangeWalletPasswordNewRootkey(t *testing.T) {
 	t.Parallel()
 
 	// testDir is empty, meaning wallet was not created from before.
-	testDir, err := ioutil.TempDir("", "testchangepassword")
-	require.NoError(t, err)
-	defer func() {
-		_ = os.RemoveAll(testDir)
-	}()
+	testDir := t.TempDir()
```

Comment:
Missed an instance of `io.TempFile()` in the loop below that causes this test to fail sometimes with `testing.go:1090: TempDir RemoveAll cleanup: unlinkat /tmp/TestChangeWalletPasswordNewRootkey1035071230/001/mainnet: directory not empty`